manifest.yaml

```diff
@@ -2,22 +2,26 @@ name: github.com/getoutreach/stencil-golang
 ## <<Stencil::Block(keys)>>
 type: templates,extension
 arguments:
-  terraform.datadog.monitoring.argocd.appHealth.notify:
-    type: boolean
-    description: Alert when ArgoCD health is not healthy
-  terraform.datadog.monitoring.argocd.syncStatus.notify:
-    type: boolean
-    description: Alert when ArgoCD syncStatus is not synced
-  terraform.datadog.monitoring.argocd.syncStatus.evaluationWindow:
-    type: string
-    description: The amount of time to query the ArgoCD Sync Status alert over (i.e. last_15m)
-  terraform.datadog.monitoring.argocd.appHealth.evaluationWindow:
-    type: string
-    description: The amount of time to query the ArgoCD App Health alert over (i.e. last_15m)
-  terraform.datadog.monitoring.generateSLOs:
-    type: boolean
-    default: true
-    description: Have stencil generate Datadog service level objectives based on HTTP/GRPC queries/monitors.
+  terraform.datadog.grpc.latency.thresholds.lowTraffic:
+    schema:
+      type: number
+    description: Overrides the low traffic threshold in the "grpc_latency_high" terraform module in monitoring/datadog.tf.
+  terraform.datadog.grpc.qos.thresholds.lowTraffic:
+    schema:
+      type: number
+    description: Overrides the low traffic threshold in the "grpc_latency_high" terraform module in monitoring/datadog.tf.
+  terraform.datadog.http.latency.thresholds.lowTraffic:
+    schema:
+      type: number
+    description: Overrides the low traffic threshold in the "http_latency_high" terraform module in monitoring/datadog.tf.
+  terraform.datadog.http.percentiles.lowTraffic:
+    schema:
+      type: number
+    description: Overrides the low traffic percentile in the "http_success_rate_low" terraform module in monitoring/datadog.tf.
+  terraform.datadog.podRestart.thresholds.lowCount:
+    schema:
+      type: number
+    description: Overrides the pod restarts threshold in the "pod_restarts" terraform module in monitoring/datadog.tf.
```

Comment:
question: are these still here because they're referenced in other Stencil modules?

Reply:
they're used in the canary stuff in jsonnet

Reply:
will add a comment

Reply:
At some point we should move these values to a more relevant fully qualified configuration option name.

Reply:
I strongly agree.